tests/time_tests/src/timetests/timetest_infer.cpp

```diff
@@ -50,6 +52,14 @@ int runPipeline(const std::string &model, const std::string &device) {
     {
       SCOPED_TIMER(first_inference);
       inferRequest = exeNetwork.CreateInferRequest();
+
+      {
+        SCOPED_TIMER(fill_inputs)
```

Comment:
@asomsiko need to discuss should `fill_inputs` step be included in `first_inference` or not

Reply:
I'm OK with current implementation